index.js

```diff
@@ -0,0 +1,108 @@
+/* eslint-disable no-sync */
+const fs = require('fs');
+const path = require('path');
+const chalk = require('chalk');
+const Funnel = require('broccoli-funnel');
+const mergeTrees = require('broccoli-merge-trees');
+const prettyBytes = require('pretty-bytes');
+const workboxBuild = require('workbox-build');
+const workboxBuildPkg = require('workbox-build/package.json');
+const debug = require('debug')('ember-cli:workbox');
+const { red, blue, yellow } = chalk;
+
+function mergeOptions(options, defaultOptions) {
+	for (const option in defaultOptions) {
+		if (!options.hasOwnProperty(option)) {
+			options[option] = defaultOptions[option];
+		}
+	}
+
+	return options;
+}
+
+function removeDir(dir) {
```

Comment:
Maybe [fs-extra](https://github.com/jprichardson/node-fs-extra) or [rimraf](https://github.com/isaacs/rimraf) to recursively delete dirs?